replayer/frame.cpp

```diff
@@ -198,15 +225,25 @@ std::ostream& replayer::operator<<(
     }
   }
 
+  // Read the rest of frame
   writeTail(out, o.actions, o.resources, o.bullets);
 
   out << " " << o.reward << " " << o.is_terminal;
   return out;
 }
 
 std::istream& replayer::operator>>(std::istream& in, replayer::Frame& o) {
-  int nPlayer;
+  int nPlayer, creep_map_size;
+
+  // Pack the creep map, since you can't serialize vec<bool>
```

Comment:
Yes, another reason to use `std::vector<int8_t>` everywhere. This functions also reads instead of packs, no?